### PR TITLE
 krakenFutures module extension :  resubmit 

### PR DESCRIPTION
**Pull request by CNiemans**
this is an old version , already coded, I am resubmitting it into testing

### CODE REVIEW

**Comment by samgermain** [comment]
Duplicate of #15133

File: README.md [hunk 1]
```diff
@@ -22,9 +22,17 @@ Current feature list:
 
 ## Sponsored Promotion
 
-Want this place? Contact us: info@ccxt.trade
+Want this place? Contact us: info@ccxt.trade
```
Comment (samgermain) [inline README.md:25]: remove this trailing space

**Comment by samgermain** [comment]
@CNiemans The title of this PR does not match the changes made within this PR. Can you summarize the purpose of this PR please?

Thank you

**Comment by CNiemans** [comment]
hello , @kroitor  , I resubmitted a js file but here the build is in python  - does this need to be translated ?

**Comment by kroitor** [comment]
> hello , @kroitor , I resubmitted a js file but here the build is in python - does this need to be translated ?

Hey! First of all, thanks for your involvement!
Unfortunately, yes, we can transpile JS > Py, but not the opposite yet:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#multilanguage-support

Technically, it should be easy to convert to JS, since CCXT integrations are transpiled line-to-line (if you compare JS code against Python code of any of the exchanges already listed in CCXT). We will do our best to assist. Will try to restore the old JS file asap.

**Comment by ttodua** [comment]
Guys, I think there was some misunderstanding. @samgermain was who had to re-submit the PR (after accidental merge & revert), however, I think that @CNiemans though that he had to re-submit it. 

Here is the PR open: https://github.com/ccxt/ccxt/pull/15133/ by Sam, so we can wait its merge, so I'll close this current topic (as this is not actual PR). let me know if any other thoughts
